composer.json

```diff
@@ -110,7 +110,7 @@
         "behat-selenium": "./ci/qa/behat-selenium",
         "test": "./ci/qa/phpunit",
         "license-headers": "vendor/bin/docheader check src/",
-        "yarn-audit": "yarn audit --groups dependencies optionalDependencies",
+        "yarn-audit": "yarn audit",
```

Comment:
Removing the audit restrictions is no problem, now that we are installing latest and greatest of the Webpack Encore packge (and its dependencies)